### PR TITLE
Add javadocs in the Presto methods

### PR DESCRIPTION
**Pull request by andreyliporage**
# Pull Request Merge Checklist
           
Task backlog identifier related to this PR: ---

---

## As a DEVELOPER, I confirm that this PR **doesn't contain**:
- [ ] Code lines with syntax errors
- [ ] Methods and variables names that aren't clear or doesn't mean anything
- [ ] Code that is out of the standard (Example: **PEP-8** for Python code)
- [ ] Files with **imports** errors or without any utility
- [ ] **TODOs** left behind
- [ ] Tests that are being skipped
- [ ] Methods placed on incorrect files or folders (Example: methods that are local, but should be placed on /utils)
- [ ] Functions, methods or API responses that are publicly returning sensitive information

## As a DEVELOPER, I confirm that I:
- [ ] Followed the Simbiose's standard for branch and commit naming
- [ ] Concluded the task and achieved the goals according to the task planning I defined before starting to code
- [ ] Added all necessary database changes to the [SQL Migrations](https://github.com/s1mbi0se/sql-migrations)
- [ ] Created tests for every new flow and method on this Pull Request
- [ ] Added the necessary DEBUG logs
- [ ] Checked my code using the Simbiose's Corner Cases Guide
- [ ] Merged the **develop** branch to this PR/branch before sending the PR for review
- [ ] Solved all possible merge conflicts before sending the PR for review
- [ ] Corrected all the problems and code smells identified by the quality tools
- [ ] Manually executed the necessary tests locally to make sure my changes didn't break anything
- [ ] All unitary and integration tests ran by the CI system are passing
- [ ] Check the performance of my changes or additions and they didn't make the system slower (I profiled the code)
- [ ] Updated the necessary documentation (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [ ] Made the self-review of the code before sending the PR for review
- [ ] Sent this PR for a peer-review
- [ ] Made all the corrections appointed during the peer-review
- [ ] Confirmed that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [ ] Checked all GitHub Actions related to this PR that can block my merge

---  

## As the TEAM MANAGER, I confirm that I:
- [ ] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [ ] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [ ] Checked if there is no possible merge conflicts before reviewing the PR
- [ ] Checked if the purpose of the task was reached with this PR
- [ ] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [ ] Checked if all related PRs were created when other components were affected by this task (e.g. database changes on [SQL Migrations](https://github.com/s1mbi0se/sql-migrations))
- [ ] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [ ] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [ ] Checked that code coverage for this PR is higher than 80%.
- [ ] Checked that there are no problems and code smells identified by the quality and security tools
- [ ] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [ ] Checked that all classes, methods and variables on this PR follow the Simbiose’s pattern for this programming language
- [ ] Checked that all unitary and integration tests ran by the CI system are passing
- [ ] Checked that there are no performance issues created by these changes or additions
- [ ] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [ ] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [ ] Checked all GitHub Actions related to this PR that can block the merge

---  

## As the QUALITY RESPONSIBLE, I confirm that I:
- [ ] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [ ] Checked if there is no possible merge conflicts before reviewing the PR
- [ ] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [ ] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [ ] Checked that code coverage for this PR is higher than 80%.
- [ ] Checked that there are no problems and code smells identified by the quality and security tools
- [ ] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [ ] Checked that all classes, methods and variables on this PR follow the Simbiose’s code standard for this programming language
- [ ] Checked that all unitary and integration tests ran by the CI system are passing
- [ ] Checked that there are no performance issues created by these changes or additions
- [ ] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [ ] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [ ] Checked all GitHub Actions related to this PR that can block the merge

## Important links:
1. [Simbiose Quality Guidelines](https://github.com/s1mbi0se/company/wiki/Quality-Topics#table-of-contents)
2. [Metadata Database Modeling](https://docs.google.com/spreadsheets/d/1Y8_k5xV86kTjgfjbi-PGsJcubUKhWlT2x8ecpnMlyR0/edit?usp=sharing)
3. [Simbiose's Corner Cases Guide](https://www.simbioseventures.com/documentos-e-guias/procedimentos-e-checklists/corner-cases-guide)

### CODE REVIEW

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 1]
```diff
@@ -354,6 +354,21 @@ public synchronized boolean isSubmissionFinished()
             return dispatchManager.waitForDispatched(queryId);
         }
 
+        /**
+         * Verifies the token of the query and returns the result.
+         * <p>
+         * Checks whether the token is the last one or the next one, comparing the value
+         * of last token and the value expected.
+         * At the end, results of the query is returned. Containing the token, the uriInfo
+         * containing the metadata and the queued time and elapsed time.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:363]: it's more than the queued and elapsed time. reference this to "query statistics"

File: presto-main/src/main/java/io/prestosql/execution/SqlTaskExecution.java [hunk 2]
```diff
@@ -603,6 +603,13 @@ private DriverStats getDriverStats()
         }
     }
 
+    /**
+     * Lists all plans nodes id for a noMoreSplits condition.
+     * <p>
+     *
+     *
+     * @return a Set of running node ids.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/execution/SqlTaskExecution.java:611]:
```suggestion
     * @return a Set of plan node ids.
```

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 3]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the raw input data size in bytes.
+     */
     @JsonProperty
     public DataSize getRawInputDataSize()
     {
         return rawInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the input positions of the raw data of the pipeline stats.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:398]: It doesn't say raw data, it's the raw input positions of the PipelineStats.

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 4]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the processed input positions.
+     */
     @JsonProperty
     public long getProcessedInputPositions()
     {
         return processedInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link Duration} that contains the input read time.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:431]:
```suggestion
     * Returns an object of type {@link Duration} that contains the physical input read time.
```

File: presto-main/src/main/java/io/prestosql/operator/TaskContext.java [hunk 5]
```diff
@@ -354,6 +354,11 @@ public CounterStat getOutputPositions()
         return stat;
     }
 
+    /**
+     * Returns an object that contains the duration between start and finish an object.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/operator/TaskContext.java:358]:
```suggestion
     * Returns an object that contains the duration between start and finish of a Presto Task.
```

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 6]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the raw input data size in bytes.
+     */
     @JsonProperty
     public DataSize getRawInputDataSize()
     {
         return rawInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the input positions of the raw input positions of the PipelineStats.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:398]:
```suggestion
     * Returns a property that represents the raw input positions of the PipelineStats.
```

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 7]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the output positions.
+     */
     @JsonProperty
     public long getOutputPositions()
     {
         return outputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the physical written data size.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:464]: missing the unit

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 8]
```diff
@@ -354,6 +354,21 @@ public synchronized boolean isSubmissionFinished()
             return dispatchManager.waitForDispatched(queryId);
         }
 
+        /**
+         * Verifies the token of the query and returns the result.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:358]: Validate the query token to return the result.

File: presto-main/src/main/java/io/prestosql/operator/TaskContext.java [hunk 9]
```diff
@@ -354,6 +354,11 @@ public CounterStat getOutputPositions()
         return stat;
     }
 
+    /**
+     * Returns an object that contains the duration between start and finish of a Presto Task.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/TaskContext.java:358]: Returns an object that contains the duration of a Presto Task.

File: presto-main/src/main/java/io/prestosql/operator/TaskContext.java [hunk 10]
```diff
@@ -368,6 +373,14 @@ public Duration getFullGcTime()
         return new Duration(max(0, endFullGcTimeNanos - startFullGcTimeNanos), NANOSECONDS);
     }
 
+    /**
+     * verifies how many objects were moved to the Garbage Collector.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/TaskContext.java:377]: This method isn't verifying anything. It's just returning the calculated value. So the description should be:
"Returns how many objects were moved to the Garbage Collector in the task context."

File: presto-main/src/main/java/io/prestosql/operator/TaskContext.java [hunk 11]
```diff
@@ -368,6 +373,14 @@ public Duration getFullGcTime()
...
+     * <p>
+     * Verifies how many objects were started and how many objects were finished and returns
+     * the total that were moved to the Garbage Collector.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/TaskContext.java:380]: "Check how many objects were instantiated and removed in the task context and then calculates the total quantity of objects that were collected by the Garbage Collector."

File: presto-main/src/main/java/io/prestosql/operator/TaskContext.java [hunk 12]
```diff
@@ -368,6 +373,14 @@ public Duration getFullGcTime()
...
+     *
+     * @return a int that represents the objects that were be moved to the Garbage Collector.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/TaskContext.java:382]: "the quantity of objects collected by the Garbage Collector"

File: presto-main/src/main/java/io/prestosql/execution/SqlTaskExecution.java [hunk 13]
```diff
@@ -603,6 +603,11 @@ private DriverStats getDriverStats()
         }
     }
 
+    /**
+     * Lists all plans nodes id for a noMoreSplits condition.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/execution/SqlTaskExecution.java:607]: A "noMoreSplits condition" means nothing to me. Please, describe it better, I would do something like this:
"List all plans node identifiers that [...]."

On the [...] I will explain what this "noMoreCondition" means. As a developer that do not know Presto, I have no idea of what you are talking about, so I can't understand the method purpose.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 14]
```diff
@@ -354,6 +354,21 @@ public synchronized boolean isSubmissionFinished()
...
+         * Checks whether the token is the last one or the next one, comparing the value
+         * of last token and the value expected.
+         * At the end, results of the query is returned. Containing the token, the uriInfo
+         * containing the metadata and the query statistics.
+         *
+         * @param token generated token.
+         * @param uriInfo an object containing the uri metadata.
+         *
+         * @return the next token, the uri metadata and the informations about the query.
+         * @throws WebApplicationException If the token wouldn't be the last one or
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:369]: "if the token isn't the last one or the dispatchInfo is empty."

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 15]
```diff
@@ -354,6 +354,21 @@ public synchronized boolean isSubmissionFinished()
...
+         * <p>
+         * Checks whether the token is the last one or the next one, comparing the value
+         * of last token and the value expected.
+         * At the end, results of the query is returned. Containing the token, the uriInfo
+         * containing the metadata and the query statistics.
+         *
+         * @param token generated token.
+         * @param uriInfo an object containing the uri metadata.
+         *
+         * @return the next token, the uri metadata and the informations about the query.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:368]: Is this correct?
In my opinion this method is returning only a QueryResults object (you may link to this class)

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 16]
```diff
@@ -354,6 +354,21 @@ public synchronized boolean isSubmissionFinished()
...
 
+        /**
+         * Verifies the token of the query and returns the result.
+         * <p>
+         * Checks whether the token is the last one or the next one, comparing the value
+         * of last token and the value expected.
+         * At the end, results of the query is returned. Containing the token, the uriInfo
+         * containing the metadata and the query statistics.
+         *
+         * @param token generated token.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:365]: What token is this? Generated for why?
I can not understand what it is and its purpose

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 17]
```diff
@@ -354,6 +354,21 @@ public synchronized boolean isSubmissionFinished()
             return dispatchManager.waitForDispatched(queryId);
         }
 
+        /**
+         * Verifies the token of the query and returns the result.
+         * <p>
+         * Checks whether the token is the last one or the next one, comparing the value
+         * of last token and the value expected.
+         * At the end, results of the query is returned. Containing the token, the uriInfo
+         * containing the metadata and the query statistics.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:363]: I would improve this description:
"Validates if the token is the last one or the next one and if the query was abandoned.  If not,throws a {@link WebApplicationException}. Otherwise, create the query results to be returned."

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 18]
```diff
@@ -344,18 +344,33 @@ public boolean isFullyBlocked()
         return blockedReasons;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the physical input data size in bytes.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:348]: It's not clear to me what is this "physical input data". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 19]
```diff
@@ -344,18 +344,33 @@ public boolean isFullyBlocked()
...
+     * @return the size of the input data in bytes.
+     */
     @JsonProperty
     public DataSize getPhysicalInputDataSize()
     {
         return physicalInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the physical input positions of the PipelineStats.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:359]: It's not clear to me what is this "physical input positions". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 20]
```diff
@@ -344,18 +344,33 @@ public boolean isFullyBlocked()
...
+     * @return the physical input positions.
+     */
     @JsonProperty
     public long getPhysicalInputPositions()
     {
         return physicalInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the internal network input data size in bytes.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:370]: It's not clear to me what is this "internal network input". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 21]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
         return internalNetworkInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the raw input data size in bytes.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:387]: It's not clear to me what is this "raw input data". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 22]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the raw input data size in bytes.
+     */
     @JsonProperty
     public DataSize getRawInputDataSize()
     {
         return rawInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the raw input positions of the PipelineStats.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:398]: It's not clear to me what is this "raw input positions". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 23]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the raw input positions.
+     */
     @JsonProperty
     public long getRawInputPositions()
     {
         return rawInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the processed input data size in bytes.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:409]: It's not clear to me what is this "processed input data". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 24]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the processed input data size in bytes.
+     */
     @JsonProperty
     public DataSize getProcessedInputDataSize()
     {
         return processedInputDataSize;
     }
 
+    /**
+     * Returns a property that represents the processed input positions of the PipelineStats.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:420]: It's not clear to me what is this "processed input positions". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 25]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the processed input positions.
+     */
     @JsonProperty
     public long getProcessedInputPositions()
     {
         return processedInputPositions;
     }
 
+    /**
+     * Returns an object of type {@link Duration} that contains the physical input read time.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:431]: It's not clear to me what is this "physical input read time". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 26]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the physical input read time.
+     */
     @JsonProperty
     public Duration getPhysicalInputReadTime()
     {
         return physicalInputReadTime;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the output data size in bytes.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:442]: It's not clear to me what is this "output data size". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 27]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the output data size.
+     */
     @JsonProperty
     public DataSize getOutputDataSize()
     {
         return outputDataSize;
     }
 
+    /**
+     * Returns a property that represents the output positions of the PipelineStats.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:453]: It's not clear to me what is this "output positions". Is any other way to make it clear for the reader?

File: presto-main/src/main/java/io/prestosql/operator/PipelineStats.java [hunk 28]
```diff
@@ -368,48 +383,88 @@ public long getInternalNetworkInputPositions()
...
+     * @return the output positions.
+     */
     @JsonProperty
     public long getOutputPositions()
     {
         return outputPositions;
     }
 
+    /**
+     * Returns an object of type {@link DataSize} that contains the physical written data size in bytes.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/operator/PipelineStats.java:464]: It's not clear to me what is this "physical written data". Is any other way to make it clear for the reader?
